### ISSUE TITLE
A10 - (NOLICENSE)# is breaking check_config_mode behavior

### ISSUE THREAD

**Issue by jjfaulk1967** [ktbyers/netmiko#1993]
I am trying to use send_config_set to send a list of commands to an A10 virtual appliance.  This is the error I am getting.

ValueError: Failed to exit configuration mode

I see that the A10 is listed as an experimental platform.  So I am willing to accept that netmiko doesn't fully support the A10 platform.  And I am willing to find another method. But should it be working?

thanks

**Comment by jjfaulk1967** [comment]
I have narrowed down the problem.  It has to do with not going into configuration mode.  In particular I think it has something to do with the prompt (but not sure).

When I SSH directly to the A10 virtual appliance, then enable and change to the active partition, I get this prompt.

LBR1[PROD-EXT](NOLICENSE)#

The prompt after going into configuration mode looks like this:

LBR1[PROD-EXT](config)(NOLICENSE)#

When I execute my script, I get the same prompt before going into configuration mode. 

LBR1[PROD-EXT](NOLICENSE)#

But when it attempts to send the command it gets this error.

LBR1[PROD-EXT](NOLICENSE)#
slb virtual-server XYZ411.TEST.COM-EXT 10.1.1.42 /32
                           ^
% Unrecognized command.Invalid input detected at '^' marker.

As you can see, it never gets the prompt LBR1[PROD-EXT](config)(NOLICENSE)# showing that it is in configuration mode. Therefore the command is invalid.

**Comment by ktbyers** [comment]
Okay, I think that `NOLICENSE)#` is breaking things. Is that rare/unusual for A10?

Basically, Netmiko is looking for `)#` to determine config mode so it thinks it is in config mode when the NOLICENSE)# is present.

**Comment by jjfaulk1967** [comment]
It's has the NOLICENSE because I am testing my script on an A10 VM in GNS3.

**Comment by ktbyers** [comment]
Yes, we will have to create a special `check_config_mode()` to handle this in the A10 driver.

**Comment by ktbyers** [comment]
Fixed here:

https://github.com/ktbyers/netmiko/pull/3611
